schemas/aboutUs.js

```diff
@@ -0,0 +1,32 @@
+export default {
+    name: "aboutUs",
```

Comment:
We should probably have a schema for `staticPage` instead of `aboutUs`. Then we can use the same schema for multiple pages.